kafka-ui-react-app/src/components/Topics/Details/Messages/Messages.tsx

```diff
@@ -219,12 +219,11 @@ const Messages: React.FC<Props> = ({
       )
       .map((p) => `${p.partition}::${p.offset}`);
 
-    setQueryParams({
+    fetchTopicMessages(clusterName, topicName, {
```

Comment:
I think we need to update our useEffect instead
```diff
React.useEffect(
  () => {
    fetchTopicMessages(clusterName, topicName, queryParams);
  }, 
-  [],
+ [clusterName, topicName, queryParams]
);
```

Reply:
@workshur the point here is that we're not changing query params to keep current seekType from the search form. I don't see how the changes you're proposing will help. Could you please explain?